### DISCUSSION TITLE
Example of usage with Nuxt3 Framework

### DISCUSSION THREAD

**Discussion by xhhcyh** [module-federation/module-federation-examples#2041] [Q&A]
![image](https://user-images.githubusercontent.com/104113677/175453338-9e12ca70-5cee-436a-9178-486740835e11.png)

Nuxt v3 stable release is on the way. 
So, is it possible to show some examples of using Nuxt3 Framework？
Local and remote modules can be both used with Nuxt3.

**Comment by ScriptedAlchemy** [comment]
If you want to start a branch and try setting up a Nuxt sample app, I can help get it working. 

I only know react so I can't kickstart a vue app. But if ones more or less setup, I can get webpack to work. 

Likely challenge will be SSR since node doesn't know how to do Federation out the box. We have written plugins for node tho so it's a nonissue

**Comment by justwiebe** [comment]
Has anybody managed to do this yet?
